### PR TITLE
Show RARBG error code next to error message

### PR DESCRIPTION
**Pull request by fernandog**
and Show "Too many requests per second!" as INFO
@duramato

### CODE REVIEW

**Comment by p0psicles** [comment]
I say go, if you can also get an approval of @medariox or @labrys merge it.

**Comment by p0psicles** [comment]
Approved

File: sickbeard/providers/rarbg.py [hunk 1]
```diff
@@ -130,11 +130,16 @@ def search(self, search_strings, age=0, ep_obj=None):  # pylint: disable=too-man
                     continue
 
                 error = data.get("error")
-                error_code = data.get("error_code")
+                error_code = try_int(data.get("error_code"))
```
Comment (labrys) [inline sickbeard/providers/rarbg.py]: Does this need to be converted to int?  If its a string compare it as a string.

Comment (p0psicles) [inline sickbeard/providers/rarbg.py]: @fernandog can you check if removing the try_int works? The other should then also be changed to if error_code not in ('20', '14'). but that should not be a problem.

Comment (fernandog) [inline sickbeard/providers/rarbg.py]: ......

**Comment by labrys** [comment]
Approved

**Comment by p0psicles** [comment]
@pymedusa/developers anyone who can test this with rarbg?

**Comment by labrys** [comment]
@p0psicles tested good here:
```
14:03:26 INFO::SEARCHQUEUE-BACKLOG-121361 :: [Rarbg] :: [43b42f6] Too many requests per second. Maximum requests allowed are 1req/2sec Please try again later!. Error code: 5
```

**Pull request merged**